erp/export/create_schema.py

```diff
@@ -0,0 +1,93 @@
+from frictionless import Schema, Field
+
+from erp import schema
+from erp.export.models import ETALAB_SCHEMA_FIELDS
+from erp.schema import FIELDS
+
+
+def generate_schema(base="base-schema.json", outfile="schema.json"):
+    table_schema = Schema(base)
+    # table_schema["version"] = "0.0.1"
+
+    for field_name in ETALAB_SCHEMA_FIELDS:
+        f = FIELDS.get(field_name)
+        if not f:
+            continue
+
+        table_schema.add_field(create_field(field_name, f))
+
+    table_schema.to_json(outfile)
+
+
+def create_field(field_name, field):
+    constraints = get_constraints(field_name, field)
+    field = Field(
+        name=field_name,
+        type=map_types(field.get("type")),
+        description=field.get("help_text_ui") or field.get("description"),
```

Comment:
Side note, we should ensure always having a non-null `help_text_ui` here; how about raising an exception if it's missing? I like the idea of a self-validating schema generator